packages/lesswrong/components/messaging/NewMessageForm.tsx

```diff
@@ -19,14 +19,17 @@ export const NewMessageForm = ({classes, conversationId, templateQueries, succes
 }) => {
   const { WrappedSmartForm, Loading, Error404 } = Components
 
+  const skip = !templateQueries?.templateCommentId;
+
   const { document: template, loading: loadingTemplate } = useSingle({
     documentId: templateQueries?.templateCommentId,
     collectionName: "Comments",
     fragmentName: 'CommentsList',
-    skip: !templateQueries
+    skip
   });
   
-  if (loadingTemplate) return <Loading/>
+  // For some reason loading returns true even if we're skipping the query?
+  if (!skip && loadingTemplate) return <Loading/>
```

Comment:
This loading set to true thing is certainly annoying, do you know if we see this anywhere else?

Reply:
Checking in another random place where we both use `skip` and return `loading` (`TagContributorsList`), it does seem like `loading` also returns `true` by default when skipped.  I still haven't worked out why this behavior only manifests in the SSR context, but I'm going to merge this in since it's currently affecting users and is very unlikely to cause issues elsewhere.